tensorflow/lite/swift/Sources/Model.swift

```diff
@@ -32,6 +33,10 @@ final class Model {
     self.cModel = cModel
   }
 
+  init?(modelData: Data) {
```

Comment:
Could you add documentations? Thanks!

Reply:
Apologies for taking so long. I just noticed this message now and pushed a commit with the docstring for this function.